tests/integration/awslambda/test_lambda_api.py

```diff
@@ -794,6 +794,75 @@ def test_lambda_image_and_image_config_crud(
             "get-function-config-response-after-delete-imageconfig", get_function_config_response
         )
 
+    @pytest.mark.aws_validated
+    def test_lambda_image_versions(
+        self, create_lambda_function_aws, lambda_su_role, test_image, snapshot, aws_client
+    ):
+        """Test lambda versions with package type image"""
+        image = test_image("alpine")
+        repo_uri = image.rpartition(":")[0]
+        snapshot.add_transformer(snapshot.transform.regex(repo_uri, "<repo_uri>"))
+        function_name = f"test-function-{short_uid()}"
+        create_image_response = create_lambda_function_aws(
+            FunctionName=function_name,
+            Role=lambda_su_role,
+            Code={"ImageUri": image},
+            PackageType="Image",
+            Environment={"Variables": {"CUSTOM_ENV": "test"}},
+            Publish=True,
+        )
+        snapshot.match("create-image-response", create_image_response)
+        aws_client.awslambda.get_waiter("function_active_v2").wait(FunctionName=function_name)
```

Comment:
The waiter is not strictly necessary because `create_lambda_function_aws` waits but it cannot harm.

sidenote: We also have two similar fixtures `create_lambda_function_aws` and `create_lambda_function` without any comment how they differ.

Reply:
That's also copied from another test, you are right!

Reply:
For the fixtures: The first one is basically a wrapper around the boto call, with a waiter and desctruction. The second one is made to be compatible with the testutil.create_lambda_function method, as a drop in replacement.